### PR TITLE
Backwards compatibility 0.14.3 <-> 0.15

### PR DESCRIPTION
**Pull request by Kucharssim**
Fixes https://github.com/jasp-stats/jaspSem/issues/10
Fixes https://github.com/jasp-stats/jasp-test-release/issues/1240

Would be good to test this properly, I think I upgraded all options that were needed, but one never knows 😓 

Kudos to @JorisGoosen for fixing the model stuff.

### CODE REVIEW

**Review by vandenman — APPROVED** [review]
So I tested this a bit and it looks good! The master branch crashes horribly, but I wasn't able to break this PR. I created a JASP file with JASP 0.14.3, fiddled with some options, saved it, and then reloaded it with this PR (not sure if there is a better way).

**Comment by Kucharssim** [comment]
> I created a JASP file with JASP 0.14.3, fiddled with some options, saved it, and then reloaded it with this PR (not sure if there is a better way).

That's my method as well. I guess a better way would be to run some sort of QuickCheck on it as there are too many options to try manually, but 🤷  
I tried a couple of different option combos, so I think it should be relatively safe (in the worst case it's at least improving the situation, not making it worse). @TimKDJ do you also wanna try to break it? Otherwise I would just merge this.

File: inst/Upgrades.qml [hunk 1]
```diff
@@ -7,7 +7,55 @@ Upgrades
 	{
 		functionName: 		"SEMSimple"
 		newFunctionName:	"SEM"
-		fromVersion:		"0.14.2"
+		fromVersion:		"0.14.3"
```
Comment (TimKDJ) [inline inst/Upgrades.qml:10]: I thought the previous version was 0.14.1?

Comment (Kucharssim) [reply inst/Upgrades.qml:10]: There were some additional for linux see: https://github.com/jasp-stats/jasp-desktop/blob/development/Docs/development/jasp-upgrade-qml.md#migrating-from-oldschool-modules

Comment (TimKDJ) [reply inst/Upgrades.qml:10]: Ah, okay!

**Comment by TimKDJ** [comment]
If I have time next week I'll try to break it and post in jasp-test-release

**Pull request merged**